rules.d/60-infiniband.rules

```diff
@@ -0,0 +1,6 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION!="remove", SUBSYSTEM=="infiniband_verbs", IMPORT{builtin}="path_id"
+
+ENV{ID_PATH}=="?*", KERNEL=="uverbs*", SYMLINK+="infiniband/by-path/$env{ID_PATH}"
+ENV{ID_PATH}=="?*", KERNEL=="uverbs*", SYMLINK+="infiniband/by-ibdev/uverbs-$attr{ibdev}"
```

Comment:
About the last line, maybe `ATTR{ibdev}=="?*"` instead of `ENV{ID_PATH}=="?*"`?

Reply:
Done.